developers/README.md

```diff
@@ -17,6 +17,44 @@ Supported development platforms:
 * [Ubuntu](dev-environment/ubuntu.md)
 * [Apple's macOS](dev-environment/macOS.md)
 
+### Virtual Machines
+
+You can test changes to BonnyCI locally by exercising some tools defined in [hoist](www.github.com/BonnyCI/hoist). Hoist is a set of ansible playbooks that automate the deployment of a BonnyCI environment. Be sure that you have completed the steps in [Development Environment](#development-environment) before proceeding.
```

Comment:
Good call. I made some changes to `dev-environment` in #39 and updated this PR to reflect it.